src-vue/src/views/RepairView.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="fc_repair__container">
+        <h2>Repair:</h2>
```

Comment:
Add a button to close RepairView and remove from menu bar (like we have for DevView)?